lib/tmdb/keywords/keywords.ex

```diff
@@ -0,0 +1,103 @@
+
+defmodule Tmdb.Keywords do
+  @moduledoc """
+  Keywords Endpoint
+
+  ## Available functions for keywords
+  Tmdb.Keywords.find(1721) # Find a keyword and its name by keyword id
+  Tmdb.Keywords.search("fight") # Search for keywords by text
+  Tmdb.Keywords.movies(1721) # find movies by a keyword ID
+  """
+  use HTTPoison.Base
+  @doc ~S"""
+  Get the basic information for a specific keyword id.
+    ## Required Parameters
+    api_key
+
+    ## Examples
+    iex> Tmdb.Keywords.find(420)
+    %{"description" => nil, "headquarters" => nil, "homepage" => nil, "id" => 420,
```

Comment:
Example result here is incorrect. It's the result of searching for companies, not keywords.